bufrsnd/rrfs_sndp.fd/CALWXT_BOURG.f

```diff
@@ -156,9 +156,9 @@ SUBROUTINE CALWXT_BOURG(T,Q,PINT,LMHK,LM,PPT,ZINT,PTYPE)
           ELSE
 C             TRANSITION ZONE, ASSUME EQUALLY LIKELY RAIN/SNOW
 C             PICKING A RANDOM NUMBER, IF <=0.5 SNOW
-              t2=rtc() 
+              t2=secnds(0.0) 
```

Comment:
"secnds(0.0)" returns the elapsed seconds since midnight. What about the extreme case where T1 is obtained just before 0z while T2 is obtained after 0z? Will the code still work as expected?
Is GNU able to compile rtc()? Thanks!

Reply:
RTC() is not in intrinsic standard call for fortran, though it apparently is supported by Intel. I think the intention here is just to get a new seed for a random number, so I am not sure it is an issue. I could change the seed to be abs(t2 - t1) to ensure the seed is positive, but I am not sure that is required.